test/unit/Swaps.spec.ts

```diff
@@ -110,6 +110,13 @@ describe('Swaps', () => {
     expect(outboundAmount).to.equal(Swaps['UNITS_PER_CURRENCY']['LTC'] * quantity);
   });
 
+  it('should calculate 0 outbound amount for a market buy order', () => {
+    const { outboundCurrency, outboundAmount } =
+      Swaps.calculateInboundOutboundAmounts(quantity, 0, true, pairId);
```

Comment:
Should we also assert this with `Number.POSITIVE_INFINITY`?

Reply:
I went ahead and added it for good measure.

Reply:
Noice!